clearwater-cluster-manager.root/usr/share/clearwater/clearwater-cluster-manager/scripts/check_cluster_state.py

```diff
@@ -55,10 +55,10 @@ def describe_clusters():
 
     for (key, value) in cluster_values.items():
         # Check if the key relates to clustering. The clustering key has the format
-        # /clearwater[</optional site name>]/<node type>/clustering/<store type>
+        # /clearwater*[</optional site name>]/<node type>/clustering/<store type>
         key_parts = key.split('/')
 
-        if key_parts[1] == 'clearwater':
+        if key_parts[1].startswith('clearwater'):
```

Comment:
These keys were fetched by querying `/clearwater/*` so all the keys first part start with (more specifically, always equals) "clearwater".